CMakeLists.txt

```diff
@@ -3,7 +3,7 @@ project(AlcorControlAgent)
 
 # Set the version number.
 set(CMAKE_BUILD_TYPE Debug)
-set(CMAKE_CXX_STANDARD 14)
+set(CMAKE_CXX_STANDARD 17)
```

Comment:
On the surface, C++ 17 doesn't give us any benefit as ACA is not using any C++ 17 specific features. Online community suggested to move from C++ 14 to C++ 17 to take advantage of the compiler enhancement. Based on my research, I figured newer C++ compiler is better (except C++ 20 is not fully baked yet). So it is a good idea to go forward since we are doing ACA perf enhancements.